project/src/main/java/com/google/sps/data/UserFeedback.java

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.Optional;
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableList;
+
+/**
+ * Represents the feedback that users enter about the recommendations they were given.
+ */
+@AutoValue
+public abstract class UserFeedback {
+
+  /** @return a list of IDs of the places that were recommended to the user. */
+  public abstract ImmutableList<String> placesRecommendedToUser();
```

Comment:
Can just be recommendedPlaces if the name is UserFeedback, think about the call UserFeedback.recommendedPlaces(), no need for repetition.

Reply:
that's a great point, I didn't think about the context